core/services/vrf/v2/listener_v2_types.go

```diff
@@ -148,7 +168,18 @@ func (lsn *listenerV2) processBatch(
 			return errors.Wrap(err, "mark logs consumed")
 		}
 
-		maxLinkStr := bigmath.Accumulate(batch.maxLinks).String()
+		maxLink, maxEth := accumulateMaxLinkAndMaxEth(batch)
+		// check if maxLink is zero, if it is, no need to set it
+		// in the meta below
```

Comment:
This probably doesn't matter, since a 0 will not add to the reserved LINK/ETH. Should save on code complexity.